.travis.yml

```diff
@@ -1,34 +1,31 @@
-language: scala
-scala:
-  - 2.10.5
-  - 2.11.7
 jdk:
   - oraclejdk7 # openJDK crashes sometimes
-sudo: false
+
+sudo: required
```

Comment:
If we no longer need to install `pip`, does it mean we don't need `sudo` either?